saleor/graphql/core/utils/__init__.py

```diff
@@ -142,7 +144,7 @@ def validate_required_string_field(cleaned_input, field_name: str):
 def from_global_id_or_error(
     id: str, only_type: Union[ObjectType, str] = None, raise_error: bool = False
 ):
-    """Resolve database ID from global ID or raise ValidationError.
+    """Resolve database ID from global ID or raise GraphQLError.
```

Comment:
```suggestion
    """Resolve global ID or raise GraphQLError.
```
And below we could list all supported global ID formats: integers, UUID and external app IDs with some example.

Reply:
done